examples/src/app/code-editor.mjs

```diff
@@ -75,26 +79,39 @@ class CodeEditor extends TypedComponent {
         window.removeEventListener("requestedFiles", this.handleRequestedFiles);
     }
 
+    /**
+     * @param {Event} event - The event.
+     */
     handleExampleLoad(event) {
         // console.log("CodeEditor got files event", event);
         /** @type {Record<string, string>} */
+        // @ts-ignore
         const files = event.files;
         this.mergeState({ files, selectedFile: 'example.mjs' });
     }
 
+    /**
+     * @param {Event} event - The event.
+     */
+    // @ts-ignore
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
The presence of these comments, as far as I'm aware, disconnect the JSDoc block from the function below.